.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: node_js
+node_js:
+  - "0.10"
+before_script:
+  - npm install -g bower
+  - bower install
+script:
+  - ./node_modules/.bin/testee tests/output/pjs/index.html
+  - ./node_modules/.bin/testee tests/output/webpage/index.html
+  - ./node_modules/.bin/testee tests/tooltips/index.html
```

Comment:
right now testee offers no benefit over 'npm run test' b/c it uses phantomjs by default.  I've had good success running firefox via testee on travis-ci, but unfortunately, our test suites do not pass in firefox.  I tried getting this to work with chrome w/o success.  setting up a unix environment to debug what's wrong with the chrome launcher is on my list of things to do.  It would also be good to fix the tests so that they work in firefox too... being able to run the tests automatically in both browsers would be ideal.